### ISSUE TITLE
Cant run admin in a fresh meteor install

### ISSUE THREAD

**Issue by maxpowel** [yogiben/meteor-admin#234]
Hi!

    I followed the steps of the readme and since the wizzard never run I created the user using accounts-ui and, adding the roles:["admin"] and the log in. My meteor version is:
    Meteor 1.1.0.2

Then only the word "Loading" appears in the screen. 0 error messages or anything else. My steps was exactly:

    meteor add iron:router
meteor add dburles:collection-helpers
meteor add aldeed:collection2
meteor add accounts-base accounts-password
meteor add alanning:roles
meteor add twbs:bootstrap
meteor add fortawesome:fontawesome
meteor add yogiben:admin

Remove all default js, css and html files of my project.
Create admin.js with the content:

    AdminConfig = {
        collections: {
            Posts: {}
        }
    };
    
    
    Schemas = {};
    
    Posts = new Meteor.Collection('posts');
    
    Schemas.Posts = new SimpleSchema({
        title: {
            type: String,
            max: 60
        },
        content: {
            type: String,
            autoform: {
                rows: 5
            }
        },
        createdAt: {
            type: Date,
            label: 'Date',
            autoValue: function () {
                if (this.isInsert) {
                    return new Date();
                }
            }
        },
        owner: {
            type: String,
            regEx: SimpleSchema.RegEx.Id,
            autoValue: function () {
                if (this.isInsert) {
                    return Meteor.userId();
                }
            },
            autoform: {
                options: function () {
                    _.map(Meteor.users.find().fetch(), function (user) {
                        return {
                            label: user.emails[0].address,
                            value: user._id
                        };
                    });
                }
            }
        }
    });
    
    Posts.attachSchema(Schemas.Posts)
    
    
    if(Meteor.isServer){
       // Accounts.createUser({email:"maxpowel@gmail.com", username:"maxpowel", password:"123456", roles:["admin"]})
    }

And admin.html with the content:

    <body>
    {{> loginButtons}}
    </body>

Then logged in as this admin user. The message telling me that I need to be an admin user disappeared and only the word "Loading" appears. Nothing else happens.

    
Thank you

**Comment by vladejs** [comment]
I have the same problem. It's curious, because just yesterday it was working fine.

**Comment by mpowaga** [comment]
For me it works with the same meteor version as you are using. Can you create a reproduction repo?

**Comment by maxpowel** [comment]
Sure, hope this is what you want
https://github.com/maxpowel/tests

I also tried meteor-starter and it works without any problem in my computer.

**Comment by vladejs** [comment]
I tried @maxpowel's repo on meteor 1.0.3.2 and it works fine. On meteor 1.1.0.2 it doesn't work, and my app doesn't work too.

I still keep wondering what's wrong, because the problem is not "yogiben-admin not running on meteor 1.1.0.2", the problem is that it worked for some time, and suddenly it stopped working with no error, just the Loading text on /admin route.

**Comment by Ajaxsoap** [comment]
It worked on my end @maxpowel repo on meteor 1.1.0.2, you have to place /admin after localhost:3000, then create your account on the top left of the page, then voila! admin dashboard. 

Thanks,

**Comment by vladejs** [comment]
@Ajaxsoap, did that, but didn't worked for me. Can somebody help!!!

**Comment by maxpowel** [comment]
@Ajaxsoap this is exactly what I do but I only get this
![screen1](https://cloud.githubusercontent.com/assets/529550/8660915/509ea7cc-29b4-11e5-8076-e759796992c0.jpg)

**Comment by vladejs** [comment]
Ok, iam done with this. 

I've created a 2 minutes video, so you can see for your self this super strange case, here is the link:

https://drive.google.com/file/d/0B5rvWHi3jRF3dktlbG1YZGI4Z1k/view?usp=sharing

**Comment by vladejs** [comment]
No solution yet?

**Comment by mpowaga** [comment]
@maxpowel I cloned your repo and it worked for me. It seems to be browser specific and caused by lazy loader. I will try to publish a fix asap.

**Comment by maxpowel** [comment]
My browser version of chrome is: Versión 40.0.2214.111 (64-bit)
and my firefox is: 34.0
Under Ubuntu 14.04.1 LTS

I tried to enter to the admin from my android phone (nexus 4, chrome) and safari but still having the same issue.

I installed meteor and downloaded the same repo in other computers and it works perfectly in every one. But from this computer I got the same issue wherever meteor is running. This computer may be charmed
